src/interfaces/wallet.cpp

```diff
@@ -37,18 +37,14 @@ class PendingWalletTxImpl : public PendingWalletTx
 
     int64_t getVirtualSize() override { return GetVirtualTransactionSize(*m_tx); }
 
-    bool commit(WalletValueMap value_map,
+    void commit(WalletValueMap value_map,
         WalletOrderForm order_form,
         std::string from_account,
         std::string& reject_reason) override
     {
         LOCK2(cs_main, m_wallet.cs_wallet);
         CValidationState state;
-        if (!m_wallet.CommitTransaction(m_tx, std::move(value_map), std::move(order_form), std::move(from_account), m_key, g_connman.get(), state)) {
-            reject_reason = state.GetRejectReason();
-            return false;
-        }
-        return true;
+        m_wallet.CommitTransaction(m_tx, std::move(value_map), std::move(order_form), std::move(from_account), m_key, g_connman.get(), state);
```

Comment:
`CommitTransaction` only temporarily returns `true` in all cases. This is about to change soonTM. Better leave this as is for now, since we pass in a `state`, which would be confusing if the return type was `void`